src/autogluon/bench/frameworks/multimodal/exec.py

```diff
@@ -135,9 +131,8 @@ def run(
                             multimodal_dataset_registry.list_keys()
 
         benchmark_dir (str): The path to the directory where benchmarking artifacts should be saved.
-        time_limit (int): The maximum amount of time (in seconds) to spend training the predictor (default: 10).
-        presets (str): The name of the AutoGluon preset to use (default: "None").
-        hyperparameters (dict): A JSON of hyperparameters to use for training (default: None).
+        constraint (str): The resource constraint used by benchmarking during AWS mode.
```

Comment:
Maybe specify what would happen if no constraint is being passed in? i.e. what's the default constraint